athena-federation-sdk/src/test/java/com/amazonaws/athena/connector/lambda/handlers/GlueMetadataHandlerTest.java

```diff
@@ -192,38 +195,96 @@ public void doListTables()
             throws Exception
     {
         List<Table> tables = new ArrayList<>();
-        tables.add(new Table().withName("table1"));
+        tables.add(new Table().withName("table3"));
         tables.add(new Table().withName("table2"));
+        tables.add(new Table().withName("table5"));
+        tables.add(new Table().withName("table4"));
+        tables.add(new Table().withName("table1"));
```

Comment:
why the ordering change here? seems significant but there is no comment.

Reply:
added comment